<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project ver. 1.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Otwór</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Wartości</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>A11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>E8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JS9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>R6</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>111</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tolerancja dla otworu</string>
      </property>
     </widget>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Wałek</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Wartości</string>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>a11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>f7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>g6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>h13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>j6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>p6</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>79</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Tolerancja dla wałka</string>
      </property>
     </widget>
     <zorder>groupBox_6</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Cale&lt;-&gt;mm</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Cale&lt;-&gt;milimetry</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Cale na mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mm na cale</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Przelicz</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wyczyść</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>900.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Masa</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>bok a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>bok b:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>scianka c:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>długość</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kształt:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Materiał:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>48</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Rura []</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rura O</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blacha</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Aluminium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brąz</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Licz</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>60</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Wynik</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>31</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>1000.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>11</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>kg</string>
       </property>
      </widget>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>8900.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="value">
     <double>25.399999999999999</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="value">
     <double>3.142000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>266</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionInformacje"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInformacje">
   <property name="text">
    <string>Informacje</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>doubleSpinBox_10</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
